package.json

```diff
@@ -44,6 +44,7 @@
     "moment": "^2.24.0",
     "sass": "^1.22.9",
     "sass-loader": "^7.1.0",
+    "tailwindcss": "^1.1.2",
```

Comment:
You sure you wanna use this? Another css library. What about buefiy/bulma? Do you want to remove them?

Reply:
No, they are different. Bulma looks nice as a component library. 
Tailwind is a pack of styles to create own components. 

So I can just set for a block: 

* create nice shadows
* make nice red background 

etc. I think I can reduce impact by removing some classes out of it, but I really liked how easily I can create custom components with it. I hope they can live together with bulma

Reply:
I think you compare Tailwind CSS to https://bulma.io/documentation/modifiers/
But there are much more styles in Tailwind than in Bulma

Reply:
From Tailwind website: 

> Unlike many other CSS frameworks, Tailwind doesn't include any component classes like form-input, btn, card, or navbar.

Reply:
I know the concept behind tailwind. I still don't think it's a good idea to have both in the same project.

Reply:
They do not intersect, really. I would keep both. 

* Bulma for standard controls/grid/etc
* Tailwind for constructing custom components

I think I can turn off initial Tailwind classes so they won't pollute global CSS namespace, and use them only in styles (as I do it via `@apply`)